PowerEditor/src/winmain.cpp

```diff
@@ -161,7 +161,7 @@ void parseCommandLine(TCHAR * commandLine, ParamVector & paramVector) {
 	//the commandline generic_string is now a list of zero terminated strings concatenated, and the vector contains all the substrings
 }
 
-bool isInList(const TCHAR *token2Find, ParamVector & params)
+bool removeIfExists(const TCHAR *token2Find, ParamVector & params)

```

Comment:
The removal feature is rather for making next parsing easier.
I would say "isInList" is better sementically.